extensions/cornerstone/src/services/SegmentationService/SegmentationService.ts

```diff
@@ -539,13 +539,29 @@ class SegmentationService extends PubSubService {
       throw new Error(`No volume found for referencedVolumeId: ${referencedVolumeId}`);
     }
 
+    const useSharedArrayBufferValue = window.config?.useSharedArrayBuffer;
```

Comment:
You can't assume the window.config is an object - it might be a function and have gotten initialized during setup.  You will have to fetch it appropriately.  I checked and it seems that appConfig is not being passed into the constructor, so my suggestion, which @sedghi should look at is to modify ServicesManager to have a setAppConfig call that takes the application configuration, and stores it, then in the create call to the service, passes in that appConfig.  You will have to call setAppConfig in appInit, but there are other related calls, so I think that is probably the right way of doing things.